public/static/js/task.js

```diff
@@ -10,6 +10,12 @@ mciModule.controller('TaskHistoryDrawerCtrl', function($scope, $window, $locatio
     return revision.revision === $scope.task.gitspec;
   }
 
+  if (window.hasBanner) {
```

Comment:
Why are these classes necessary on the task page and are not more general?

Reply:
The places where code like this was added were pages that had parts of them covered up by the banner. They all have at least 1 component positioned in a set spot, so this code is moving that spot